src/components/interaction-designer/block-editors/EndRecordingDigitsEditor.vue

```diff
@@ -9,8 +9,8 @@
           v-model="endRecordingDigits"
           :label="'flow-builder.end-recording-by-pressing' | trans"
           :placeholder="'flow-builder.enter-value' | trans"
-          :valid-state="isValid" 
-          :data-cy='"end-recording"'/>
+          :valid-state="isValid"
+          data-cy="end-recording-digit--selector" />
```

Comment:
@pascaleli , in vue js, if you provide a simple string as value, you don't need the `:` in front of the attribute. You'd need that if you will provide some dynamic values eg: `:data-cy="`block--${blockId}`"` https://github.com/FLOIP/flow-builder/blob/feature/CORE-549/cypress--integration/src/components/interaction-designer/block/Block.vue#L6